biocrnpyler/crnlab.py

```diff
@@ -23,12 +23,18 @@ def __init__(self, name = '', **kwargs):
         self.crn = None
         self.warning_print = True
 
+        warnings.warn("CRNLab is depricated and will cease to function with future releases.")
```

Comment:
Typo here (and other places): It should be "deprecated".

Reply:
Will fix!